.github/workflows/dotnetcore.yml

```diff
@@ -18,7 +18,6 @@ on:
       - dev
 
 env:
-  TargetNet9: True
```

Comment:
I would actually keep, but rename TargetNet10, as in a few weeks/months, we'll need to add TargetNet10 ...

Reply:
Added `TargetNetNext` flag.